pkg/chain/ethereum/tbtc_test.go

```diff
@@ -279,6 +280,49 @@ func TestCalculateInactivityClaimHash(t *testing.T) {
 	)
 }
 
+func TestCalculateWalletID(t *testing.T) {
+	hexToByte32 := func(hexStr string) [32]byte {
+		if len(hexStr) != 64 {
+			t.Fatal("hex string length incorrect")
+		}
+
+		decoded, err := hex.DecodeString(hexStr)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		var result [32]byte
+		copy(result[:], decoded)
+
+		return result
+	}
+
+	xBytes := hexToByte32(
+		"9a0544440cc47779235ccb76d669590c2cd20c7e431f97e17a1093faf03291c4",
+	)
+
+	yBytes := hexToByte32(
+		"73e661a208a8a565ca1e384059bd2ff7ff6886df081ff1229250099d388c83df",
+	)
```

Comment:
Are we using an existing wallet PK here? If not, can we use one from `tbtcscan` to make sure this logic is correct?

Reply:
I copied this test data from some existing `WalletRegistry` contract test:
https://github.com/keep-network/keep-core/blob/ea48ec5f13465461f232b01e92f9a10bf0fab492/solidity/ecdsa/test/WalletRegistry.Wallets.test.ts#L29.

Reply:
Ok, makes sense!